awx/main/conf.py

```diff
@@ -765,6 +765,34 @@
     category=_('Logging'),
     category_slug='logging',
 )
+register(
+    'LOG_AGGREGATOR_BATCH_ENABLED',
+    field_class=fields.BooleanField,
+    default=False,
+    label=_('Enable batching of log messages'),
+    help_text=_('Enable batching of log messages to the external log aggregator.'),
```

Comment:
Include something about these settings not having any effect unless you are using the omhttp external logger. But don't use the word omhttp external logger because no one will know what that is. Not sure what we call it "splunk" or http?